tests/libs/superlu_dist/tests/rm_execution

```diff
@@ -13,8 +13,11 @@ rm=$RESOURCE_MANAGER
 
 testname="libs/superLU_dist"
 
+if [ ! -z "$SIMPLE_CI" ]; then
+export OMPI_MCA_btl="self,tcp"
+fi
 NODES=2
-TASKS=`tasks_count 4`
+TASKS=4
```

Comment:
Why is this change necessary? I thought `tasks_count` it the function which transparently tries to do the right thing if running in GitHub Actions?

Reply:
Initially (few months ago) this test was using `4`. Then I updated it to `tasks_count 4` which in GHA/Cirrus actually returns `2` (as the number of cores). 
The tests were failing with `2` and passed with `4`. 
Let me try one more time with `tasks_count 4`!

Reply:
With `2` the tests fail with:
`# Number of processes 2 is smaller than NPROW * NPCOL 4Number of processes 2 is smaller than NPROW * NPCOL 4`